db/schema.rb

```diff
@@ -108,7 +106,7 @@
     t.decimal "student_loan_repayment", precision: 7, scale: 2
     t.decimal "tax", precision: 7, scale: 2
     t.decimal "net_pay", precision: 7, scale: 2
-    t.index ["claim_id"], name: "index_payments_on_claim_id", unique: true
+    t.index ["claim_id"], name: "index_payments_on_claim_id"
```

Comment:
Was this change intentional?

Reply:
Nah, all I did was run the migration. I'll do it all again from cold and see if the same thing happens

Reply:
I've recreated the schema from scratch and it hasn't happened again, so I think something must've got out of sync